commit b7382b54bee489bafbeb943637bec950cb5cd699 (refs/changes/18/818/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-31T11:54:18-07:00 (2 years, 11 months ago)
    
    TOOLS-1594 headnode-joyent build failed with "mget: AssertionError: undefined (object) is required"
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/package.json b/package.json
index 8296a56..f72a187 100644
--- a/package.json
+++ b/package.json
@@ -1,11 +1,11 @@
 {
   "name": "mountain-gorilla",
   "description": "Build SDC components. Eat bamboo.",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-    "manta": "3.0.0",
+    "manta": "3.1.3",
     "smartdc": "7.3.0",
     "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#38332b1"
   },
